toolchain/lowering/lowering_function_context.h

```diff
@@ -48,13 +71,26 @@ class LoweringFunctionContext {
     return lowering_context_->GetType(type_id);
   }
 
+  // Create a synthetic block that corresponds to no SemanticsNodeBlockId. Such
+  // a block should only ever have a single predecessor, and is used when we
+  // need multiple `llvm::BasicBlock`s to model the linear control flow in a
+  // single SemanticsIR block.
+  auto CreateSyntheticBlock() -> llvm::BasicBlock*;
+
+  // Determine whether block_ is the most-recently-created synthetic block.
```

Comment:
```suggestion
  // Determine whether block_ is the most recently created synthetic block.
```

I'm declaring war on unnecessary hyphens. :-P

Reply:
Hm, https://developers.google.com/style/hyphens#compound-modifiers-before-a-noun says to hyphenate "if needed for clarity". That's not especially helpful here :) But I guess the fact that you're asking for this indicates these hyphens aren't needed for clarity.